.github/workflows/maven-publish.yml

```diff
@@ -3,14 +3,16 @@ name: Publish snapshots to maven
 on:
   workflow_dispatch:
   push:
-    branches: [
-      main
-        1.*
-        2.*
-    ]
+    branches:
+      - main
+      - 1.*
+      - 2.*
```

Comment:
can we change this to '1.3' and 2.x? 
https://github.com/opensearch-project/reporting/pull/665/files

I am not sure if this will work, but not sure, only way is to test it out i guess, but saw other repos using this syntax, I'm not sure if github workflows will be able to do regex

Reply:
It supports glob patterns: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
checked ml-commons have similar syntax and CI is working.